src/Concerns/HasFactoryResolver.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace KoalaFacade\DiamondConsole\Concerns;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+trait HasFactoryResolver
```

Comment:
what if this one using abstract class then Factory Contract will be here, so in Factory concrete will implement according factory e.g (UserFactory)

Reply:
<img width="867" alt="image" src="https://user-images.githubusercontent.com/56016477/205495414-8126e514-c688-44e2-bee6-fa9aace9bba1.png">
here example

Reply:
aite

Reply:
<img width="578" alt="image" src="https://user-images.githubusercontent.com/56016477/205495469-4a9f7355-d0c8-49ac-a37c-63cd98bd9dd7.png">

here is consumer